roles/yoda_rulesets/defaults/main.yml

```diff
@@ -19,16 +18,20 @@ core_rulesets:
 
 extra_rulesets: []
 
-# UU Module specific configuration
-enable_async_replication: 1
+# Replication configuration
+enable_async_replication: 1            # Enable replication: yes (1) or no (0)
+async_replication_jobs: 1
```

Comment:
I would recommend adding this parameter to the configuring-yoda documentation page